plugin/uuid.go

```diff
@@ -0,0 +1,28 @@
+package plugin
```

Comment:
Please keep this in the main `plugin.go` file. I don't disagree that we should think about splitting it up but I'd rather do that separately in a more structured way with only that as a goal rather than starting the split on a case-by-case basis.

Reply:
fixed, moved it to plugin.go